api/src/main/resources/org/openmrs/api/db/hibernate/User.hbm.xml

```diff
@@ -35,6 +35,8 @@
                   column="system_id" length="50"/>
 
         <property name="username" type="java.lang.String" length="50"/>
+        
+        <property name="email" type="java.lang.String" length="255"/>
```

Comment:
Add not null constraint

Reply:
Adding  not null constraint to this field result in alot of test failures example
Caused by: org.h2.jdbc.JdbcSQLException: NULL not allowed for column "EMAIL"; SQL statement:
insert into USERS (USER_ID, PERSON_ID, SYSTEM_ID, USERNAME, PASSWORD, SALT, SECRET_QUESTION, CREATOR, DATE_CREATED, CHANGED_BY, DATE_CHANGED, RETIRED, RETIRE_REASON, UUID) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [23502-196]

Reply:
Also i  checked the UserValidator and it used a matcher to verify the pattern of an email. i just reused it since org.apache.commons.validator.EmailValidator was not available, Or should i add the package and use but it instead